ci/build.py

```diff
@@ -39,6 +39,7 @@
 from itertools import chain
 from subprocess import call, check_call
 from typing import *
+from util import *
```

Comment:
Is this addition needed?  Looks like you only removed from this file.

Reply:
maybe not